CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 54 Release Notes
```

Comment:
Just NB if we don't merge this in time for the release today, you might need to move the comment